.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  stories: ["../packages/**/__stories__/*.stories.js"],
```

Comment:
i like the convention of each component having `__stories__` and `__tests__`, but interested in others' thoughts on this

Reply:
i like that! any particular reasoning behind the underscores?

Reply:
it's common when tests and documentation (i'd consider storybook documentation) are co-located with source code, so i think of it as "this is relevant to this directory, but separate from source code". it also ensures those directories come first in your editor, which is kind of nice